aiomisc/backoff.py

```diff
@@ -128,3 +134,35 @@ async def run() -> Any:
 
         return wrap
     return decorator
+
+
+def asyncretry(
+    max_tries: int,
+    exceptions: Tuple[Type[Exception], ...] = (Exception,),
+    pause: Number = 0,
+    giveup: Callable[[Exception], bool] = None,
+    statistic_name: Optional[str] = None,
+) -> ReturnType:
+    """
+    Shortcut of ``asyncbackoff(None, None, 0, Exception)``.
+
+    In case of exception function will be called again with similar
+    arguments after ``pause`` seconds.
+
+    :param max_tries: is maximum count of execution attempts (>= 1).
```

Comment:
Why it is not allowed to have unlimited retries here?

Reply:
just type `99999999` if you're crazy enough

Reply:
Ok, made as explicit `None`.